admin-manual/installation/installation.rst

```diff
@@ -204,7 +208,14 @@ Ubuntu 14.04, which makes this step necessary.
 
 .. code:: bash
 
-   uudo pip install -U setuptools
+   sudo pip install -U setuptools
+
+If an error is returned, pip may be missing from your distribution, in which case you
+will need to install it separately:
+
+.. code:: bash
+
+   sudo apt-get install python-pip
```

Comment:
Using the version pip supplied by the package manager on Ubuntu 14.04 doesn't work (see the 'update pip' section of this file).  There is a bug in 14.04 that we have worked around by installing pip using get-pip.py